static/js/containers/App.js

```diff
@@ -177,6 +177,8 @@ class App extends React.Component<Props> {
       return <Redirect to={AUTH_REQUIRED_URL} />
     }
 
+    const isCourseUrl = pathname.startsWith(`/courses`)
```

Comment:
`react-router` should be able to handle this kind of logic, I think also because we're now starting to really branch out the UI between channel-related UI and non-channel UI, we should start trying to address some of this now before it necessitates an even bigger refactor. You could then do a `<Switch>` with the first route being for `/courses` and the second one being a default match for the rest of the site and put the corresponding toolbars I described above in each. `<Drawer/>` below would also go in the default match and then you don't need this kind of logic here anymore.

Reply:
Modified `<Switch><Route>` in `App.js` to determine what to render (`Toolbar` or `CourseToolbar`, `Drawer` or `null`)